query_optimizer/ExecutionGenerator.cpp

```diff
@@ -778,6 +778,8 @@ void ExecutionGenerator::convertCopyFrom(
 
   insert_destination_proto->set_insert_destination_type(S::InsertDestinationType::BLOCK_POOL);
   insert_destination_proto->set_relation_id(output_relation->getID());
+  insert_destination_proto->mutable_layout()->CopyFrom(
+      output_relation->getDefaultStorageBlockLayout().getDescription());
```

Comment:
Talked with @saketj about this- he said it was more efficient because there'd be one less copy operation. IMO seems like a trivial difference unless the code were to be in a tight loop.